pyproject.toml

```diff
@@ -26,11 +27,24 @@ Home = "https://archlinux.org"
 Documentation = "https://archinstall.readthedocs.io/"
 Source = "https://github.com/archlinux/archinstall"
 
+[project.optional-dependencies]
+doc = ["sphinx"]
+
 [project.scripts]
 archinstall = "archinstall:run_as_a_module"
 
-[project.optional-dependencies]
-doc = ["sphinx"]
+[tool.setuptools]
+packages = ["archinstall", "profiles", "examples"]
+
+[tool.setuptools.package-data]
+archinstall = [
+    "examples/*.py",
+    "profiles/*.py",
+    "profiles/applications/*.py"
+]
+
+[tool.setuptools.dynamic]
```

Comment:
@Torxed this section was already defined on line 46

Reply:
Yea the merge from `master` did'nt take that into account so it created double entries. I manually consolidated them.